rest_framework_json_api/renderers.py

```diff
@@ -61,10 +63,16 @@ def extract_attributes(cls, fields, resource):
                 continue
             # Skip fields with relations
             if isinstance(
-                    field, (relations.RelatedField, relations.ManyRelatedField, BaseSerializer)
+                    field, (relations.RelatedField, relations.ManyRelatedField)
             ):
                 continue
 
+            if isinstance(field, BaseSerializer):
+                if rendering_strategy == RELATIONS_RENDERING_STRATEGY:
+                    continue
+                elif rendering_strategy == ATTRIBUTE_RENDERING_STRATEGY:
+                    pass
```

Comment:
I'd prefer we don't have an empty elif clause but comment above why in a certain condition rendering of a field is skipped.